src/debugger/debugSessionBase.ts

```diff
@@ -97,6 +99,8 @@ export abstract class DebugSessionBase extends LoggingDebugSession {
 
     protected abstract establishDebugSession(attachArgs: IAttachRequestArgs, resolve?: (value?: void | PromiseLike<void> | undefined) => void): void;
 
+    protected abstract createJsDebugDebuggingConfiguration(attachArgs: IAttachRequestArgs): any;
```

Comment:
My opinion that we have an adapter from one array of arguments into another, maybe it is better to name it like this? For example: `adaptToJsDebugConfig` or maybe it will be better to create `JsDebugConfigAdapter`. Thoughts?